.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: rust
+rust:
+  - stable
+  - beta
+  - nightly
```

Comment:
We use all three because they suggest it here: https://docs.travis-ci.com/user/languages/rust/
We need at least stable and beta for now pending async/await support.